contents/docs/user-guides/actions.md

```diff
@@ -4,109 +4,93 @@ sidebar: Docs
 showTitle: true
 ---
 
-> Would you rather go through a tutorial about actions instead of this feature reference? Check out our [Complete Guide to Event Tracking](/docs/tutorials/actions#sorting-through-your-events-with-actions).
+> Would you rather go through a tutorial about actions instead of this feature reference? Check out our [Complete Guide to Event Tracking](/docs/tutorials/event-tracking-guide#sorting-through-your-events-with-actions).
 
-Actions are PostHog’s way of easily cleaning up a large amount of Event data.
+Actions are PostHog’s way of easily cleaning up a large amount of event data.
 
-Actions consist of one or more events that you have decided to put into a manually-labeled bucket. They're used in PostHog's Insights, as well as in Feature Flags.
+Actions consist of one or more events that you have decided to put into a deliberately-labeled bucket. They're used in insights and dashboards.
 
 For example, a typical action might be one of the following:
 
-* ‘Sign up button – clicked’
-* ‘Profile page – viewed’
-* ‘Post submit – clicked’
-* ‘Pricing page – viewed’
-* ‘Watch movie – clicked’
+* ‘Clicked sign up button’
+* ‘User created account’
+* ‘Viewed pricing page’
+* ‘Submitted billing form’
 
-### Demo Video
 
-If you'd like to watch a video about our Actions feature, check out our demo video below. It is set to start on the Actions section:
+To access the Actions page, go to the Event section on the left-hand navigation sidebar and then click on the Actions tab.
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/aUILrrrlu50?start=40" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+## Creating actions
 
-## Accessing Actions
```

Comment:
Is there any chance that some of these headers are linked to from anywhere else in the docs?

Reply:
Not to my knowledge, but I could be missing something